src/api/urls.py

```diff
@@ -22,67 +23,60 @@
+                    r"(?P<language_code>[-\w]+)/sent_push_notifications/?$",
                     sent_push_notifications,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback$",
+                    r"(?P<language_code>[-\w]+)/feedback/?$",
                     legacy_feedback_endpoint.legacy_feedback_endpoint,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/page$",
+                    r"(?P<language_code>[-\w]+)/feedback/page/?$",
                     page_feedback.page_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/event$",
+                    r"(?P<language_code>[-\w]+)/feedback/event/?$",
                     event_feedback.event_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/imprint-page$",
+                    r"(?P<language_code>[-\w]+)/feedback/imprint-page/?$",
                     imprint_page_feedback.imprint_page_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/categories$",
+                    r"(?P<language_code>[-\w]+)/feedback/categories/?$",
                     region_feedback.region_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/search$",
+                    r"(?P<language_code>[-\w]+)/feedback/search/?$",
                     search_result_feedback.search_result_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/extras$",
+                    r"(?P<language_code>[-\w]+)/feedback/(extras|offers)/?$",
                     offer_list_feedback.offer_list_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/offers$",
-                    offer_list_feedback.offer_list_feedback,
-                ),
-                url(
-                    r"(?P<language_code>[-\w]+)/feedback/extra$",
-                    offer_feedback.offer_feedback,
-                ),
-                url(
-                    r"(?P<language_code>[-\w]+)/feedback/offer$",
+                    r"(?P<language_code>[-\w]+)/feedback/(extra|offer)/?$",
                     offer_feedback.offer_feedback,
                 ),
                 url(
-                    r"(?P<language_code>[-\w]+)/feedback/events$",
+                    r"(?P<language_code>[-\w]+)/feedback/events/?$",
                     event_list_feedback.event_list_feedback,
                 ),
-                url(r"(?P<language_code>[-\w]+)/pages/$", pages),
-                url(r"(?P<language_code>[-\w]+)/offers/$", offers),
-                url(r"(?P<language_code>[-\w]+)/page/$", single_page),
+                url(r"(?P<language_code>[-\w]+)/pages/?$", pages),
+                url(r"(?P<language_code>[-\w]+)/offers/?$", offers),
```

Comment:
```suggestion
                url(r"(?P<language_code>[-\w]+)/(extras|offers)/?$", offers),
```
Should we add both urls here as well?

Reply:
yes